From 43ca4cb2cb81e4107710e8a8c48df31d022a1136 Mon Sep 17 00:00:00 2001
From: Laura Barcziova <lbarczio@redhat.com>
Date: Wed, 9 Mar 2022 07:50:29 +0100
Subject: [PATCH] Packit: build SRPMs in Copr

Add srpm_build_deps key to the Packit config to specify needed dependencies for SRPM build
and indicate to build SRPM in Copr.

(cherry picked from commit d15e1a29e3aab04ee79d5e3ec8e1e65fca78e165)

Related: #2155516
---
 .packit.yml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/.packit.yml b/.packit.yml
index ce8782aae2..e1cbea3b3a 100644
--- a/.packit.yml
+++ b/.packit.yml
@@ -13,6 +13,7 @@ downstream_package_name: systemd
 # `git describe` returns in systemd's case 'v245-xxx' which breaks RPM version
 # detection (that expects 245-xxxx'). Let's tweak the version string accordingly
 upstream_tag_template: "v{version}"
+srpm_build_deps: []
 
 actions:
   post-upstream-clone:
